lib/ui/window.dart

```diff
@@ -172,18 +172,14 @@ enum AppLifecycleState {
   ///
   /// When the application is in this state, the engine will not call the
   /// [Window.onBeginFrame] and [Window.onDrawFrame] callbacks.
-  ///
-  /// Android apps in this state should assume that they may enter the
-  /// [suspending] state at any time.
   paused,
 
-  /// The application will be suspended momentarily.
-  ///
-  /// When the application is in this state, the engine will not call the
-  /// [Window.onBeginFrame] and [Window.onDrawFrame] callbacks.
+  /// The application is detached from view.
   ///
-  /// On iOS, this state is currently unused.
-  suspending,
+  /// When the application is in this state, the engine is running without
```

Comment:
I think this needs more documentation to understand what this state actually means. Can this state happen in the non add2app case?

Reply:
This can happens whenever the condition is met during engine life cycle, and yes it also happens in non add2app case. I found it weird to specifically mention add2app/non add2app in the comment, so I didn't include it. I rephrased a little bit to make it more descriptive.